tests/conftest.py

```diff
@@ -1,8 +1,8 @@
 from binascii import hexlify
 from datetime import timedelta
-from ethereum import tester
+from ethereum.tools import tester as tester
```

Comment:
the `as tester` suffix here is unnecessary and redundant I believe.  Same comment for all other places that have this import.